### ISSUE TITLE
How is this supposed to be used?

### ISSUE THREAD

**Issue by SIsilicon** [NovemberDev/godot_curvature_shader#2]
The file with .gd extension (script) contains shader code. Which would cause it to not compile correctly. Even if it's supposed to be a shader, could you give some documentation to make it easier to understand?

**Comment by NovemberDev** [comment]
I'm very sorry for the inconvenience, this is my very first asset and it has a lot of things to improve. Thank you for pointing the error with the extension out - I fixed that.

I also added a comment in the shader code. The changes will be online once the moderators approved the commit in the Godot Asset Library.

Here is a list of steps to do to get the shader to work:
1. create a MeshInstance with a Plane, subdivided by 20x20
2. assign ShaderMaterial, assign the shader to the shader slot
3. set the shader parameters to these values:

CURVATURE (default: 5.0)
CURVATURE_ACTIVE (default: true)
CURVATURE_DISTANCE (default: 25.0)

BASE_TEX (Texture for mesh)

4. Either scale the Plane or move far away with the camera to see the curvature in action

Your scene should look something like this:
![grafik](https://user-images.githubusercontent.com/46710246/71384513-0e3df700-25e2-11ea-8f9e-83658b602747.png)
